### DISCUSSION TITLE
how fix a this?

### DISCUSSION THREAD

**Discussion by SMAV12221** [n3d1117/chatgpt-telegram-bot#306] [Q&A] [answered]
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/116415774/0247e120-c865-4e1e-853c-1bef877c6029)

please help me I have a project in a week

**Comment by takada47** [comment]
влад, ты переименовал ".env.example" в ".env" ? 
и советую где ADMIN_USER_IDS и ALLOWED_TELEGRAM_USER_IDS значение после равно поставить на "*"

**Reply by SMAV12221** [reply]
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/116415774/0d839b5c-b38a-4672-aa3e-cbb18dab613d)
та же самая ошибка, тип файла только почему-то "EXEMPLE" так и должно быть?

**Comment by n3d1117** [comment, marked as answer]
Hi @SMAV12221 next time please redact your openai key and telegram token!! Anyway, make sure the file is named exactly `.env` and is in the correct folder

**Reply by SMAV12221** [reply]
![image](https://github.com/n3d1117/chatgpt-telegram-bot/assets/116415774/82940d35-3818-4ed1-a051-dae79865c8f0)
kind of like being in the right folder
